scripts/tableGenerator.js

```diff
@@ -1578,25 +1578,20 @@ function generateCart() {
         message += `${name} : ${item.quantity}Kg \n`;
     });
     if (currentOrders.length === 0) {
-        window.alert('There are no orders placed');
+        window.confirm('There are no orders placed');
         return '';
     } else {
-        window.alert(`${message}`);
+        result = window.confirm(`${message}`);
```

Comment:
Here `message` itself is a formatted string so you can directly use `window.confirm(message)`

Reply:
Not using `${message}`, is generating junk in the alert box(For eg: Hi%20,%20My%20name...)